crates/cli/src/templates/js-astgrep/workflow.yaml

```diff
@@ -17,13 +18,13 @@ nodes:
     runtime:
       type: direct
     steps:
-      - name: "Remove console.log statements"
+      - name: Remove console.log statements
         js-ast-grep:
-          js_file: "scripts/codemod.ts"
-          base_path: "."
+          js_file: scripts/codemod.js.ts
```

Comment:
Why `.js.ts`?

Reply:
Ah, that's the template file names. When the file gets created, it's actually named `codemod.ts` always: https://github.com/codemod-com/codemod/blob/3223123fad77b83b7c51b05a353d9ed7660fccbf/crates/cli/src/commands/init.rs#L436

that `.<lang>` extension prefix is just for the template file name in our source code.

Reply:
cool cool cool let's me update that